$NetBSD: patch-ab,v 1.3 2013/02/12 04:35:24 hiramatsu Exp $

Include sys/time.h to get struct timeval in all cases.

--- src/connection_base.cxx.orig	2013-02-11 10:35:50.000000000 +0000
+++ src/connection_base.cxx
@@ -24,6 +24,7 @@
 #include <cstring>
 #include <ctime>
 #include <stdexcept>
+#include <sys/time.h>
 
 #ifdef PQXX_HAVE_SYS_SELECT_H
 #include <sys/select.h>
